test/manual/src/manualTests.ts

```diff
@@ -17,37 +17,33 @@ async function main() {
     production: false,
   });
 
-  try {
```

Comment:
I think it's nice to be able to run all of the tests here - so if auth fails we can still see if sign works - maybe what was missing is some way to track "errored" status?

Reply:
Fair enoguh! I can update it to produce a summary of some sort at the end.